scripts/h2/schema-0.7.0-h2.sql

```diff
@@ -0,0 +1,290 @@
+--
```

Comment:
Once we are going to release a new version then a corresponding schema SQL file is needed, am I right?

Reply:
What is the meaning, can you please explain more?